app/assets/javascripts/initializePage.js

```diff
@@ -40,7 +40,6 @@ function callInitializers() {
 
   initializeBaseTracking();
   initializePaymentPointers();
-  initializeTouchDevice();
```

Comment:
Pulled this out because this logic is now bootstrapped in `app/javascript/packs/base.js`.